repository.json

```diff
@@ -2119,6 +2119,19 @@
 					"tags": true
 				}
 			]
+		},
+		{
+			"name": "pycparser",
+			"description": "Complete C99 parser in pure Python",
+			"author": "Eli Bendersky",
+			"issues": "https://github.com/eliben/pycparser/issues",
+			"releases": [
+				{
+					"base": "https://github.com/eliben/pycparser",
+					"python_versions": ["3.3", "3.8"],
```

Comment:
`pycparser` does not support Python 3.3, according to the README. You may find an older artifact on pypi that does, but we don't recommend adding support for 3.3 with new libraries because the 3.3 is a dead horse (now that PC properly supports 3.8).